src/chttpd/src/chttpd_prefer_header.erl

```diff
@@ -22,6 +22,11 @@
 -include_lib("couch/include/couch_db.hrl").
 
 
+-define(DEFAULT_PREFER_MINIMAL,
```

Comment:
This works in this particular case.
```
1> S = "Cache-Control, Content-Length, Content-Range,
1>      Content-Type, ETag, Server, Transfer-Encoding, Vary".
"Cache-Control, Content-Length, Content-Range, \n     Content-Type, ETag, Server, Transfer-Encoding, Vary"
2> re:split(S, "\\s*,\\s*", [trim, {return, list}]).
["Cache-Control","Content-Length","Content-Range",
 "Content-Type","ETag","Server","Transfer-Encoding","Vary"]
```

However usually splitting long strings done differently:
```
6> S2 = "Cache-Control, Content-Length, Content-Range, "
6>      "Content-Type, ETag, Server, Transfer-Encoding, Vary".
"Cache-Control, Content-Length, Content-Range, Content-Type, ETag, Server, Transfer-Encoding, Vary"
```

Reply:
Just to clarify the difference is "\n" embedded into the string.

Reply:
Thanks for the clarification, changed.